data/store/mongo/mongo_alerts.go

```diff
@@ -54,18 +54,18 @@ func (r *alertsRepo) EnsureIndexes() error {
 		{
 			Keys: bson.D{
 				{Key: "userId", Value: 1},
-				{Key: "followedId", Value: 1},
+				{Key: "followedUserId", Value: 1},
 			},
 			Options: options.Index().
 				SetUnique(true).
-				SetName("UserIdFollowedIdTypeUnique"),
+				SetName("UserIdFollowedUserIdTypeUnique"),
 		},
 	})
 }
 
 func (r *alertsRepo) filter(cfg *alerts.Config) interface{} {
 	return &alerts.Config{
-		UserID:     cfg.UserID,
-		FollowedID: cfg.FollowedID,
+		UserID:         cfg.UserID,
```

Comment:
Have we deployed this code anywhere? That is, have the previous indexes already been created and/or is the data stored with `followedId` anywhere? If so, we may have "bad data" for those environments. Not usually a big deal for QA environments, but something to be aware of.